src/app/containers/ChartbeatAnalytics/utils/index.js

```diff
@@ -35,24 +36,53 @@ export const getType = (pageType, shorthand = false) => {
       return shorthand ? 'IDX' : 'Index';
     case 'article':
       return shorthand ? 'ART' : 'New Article';
+    case 'MAP':
+      return 'article-media-asset';
+    case 'media':
+      return 'Live Radio';
     default:
       return null;
   }
 };
 
-export const buildSections = (service, type, producer, chapter) => {
+export const buildSections = ({
+  service,
+  pageType,
+  producer,
+  chapter,
+  sectionName,
+  categoryName,
+}) => {
   const addProducer = producer && service !== producer;
   const serviceCap = capitalize(service);
-  const pageType = getType(type, true);
-
-  const parts = [
-    serviceCap,
-    ...(type ? buildServiceType(serviceCap, pageType) : []),
-    ...(addProducer ? buildSectionArr(serviceCap, producer, pageType) : []),
-    ...(chapter ? buildSectionArr(serviceCap, chapter, pageType) : []),
-  ];
+  const type = getType(pageType, true);
+  const appendCategory = name => `${name}-category`;
 
-  return parts.join(', ');
+  switch (pageType) {
+    case 'MAP':
+      return [
+        serviceCap,
+        buildSectionItem(serviceCap, sectionName),
+        buildSectionItem(serviceCap, pageType),
+        buildSectionItem(buildSectionItem(serviceCap, sectionName), pageType),
+        buildSectionItem(serviceCap, appendCategory(categoryName)),
+      ].join(', ');
+    case 'media':
+      return [
+        serviceCap,
+        buildSectionItem(serviceCap, ''),
+        buildSectionItem(serviceCap, 'unknown'),
+        buildSectionItem(serviceCap, 'unknown'),
+        buildSectionItem(serviceCap, 'unknown'),
+      ].join(', ');
```

Comment:
Is this just temporary until we have values for Live Radio?

Reply:
No, this is the expected value for Live Radio.